routes/v1.js

```diff
@@ -384,7 +386,7 @@ api.declare({
 
 /** Define tasks */
 api.declare({
-  method:     'post',
+  method:     'put',
```

Comment:
this is a breaking change...  We can't add this... Well, we would have to facilitate a gradual migration...
I don't think it's worth it.

Note: the argument for using `post`  is that the resource `/task/:taskId/define` will not be created... This creates the resource `/task/:taskId`, hence calling it `put` is a bit off... Note, here I just abuse `post` to be anything that doesn't feel REST enough :)

Neither of this is more REST, both violates the principals a bit, so I don't see the benefit of changing it... Considering the overhead of fixing it everywhere..